src/Config/ConfigOverrides.php

```diff
@@ -70,10 +120,109 @@ public function loadOverrides($names) {
       }
       $overrides['config_ignore.settings']['ignored_config_entities'] = $existing_ignored;
     }
+
+    // Theme settings override.
+    $this->setLockupOverrides($names, $overrides);
+
+    // GTM overrides.
     $this->setOverridesGoogleTag($names, $overrides);
     return $overrides;
   }
 
+  /**
+   * Disable google tag manager entities when not on prod environment.
+   *
+   * @param array $names
+   *   Config names.
+   * @param array $overrides
+   *   Keyed array of config overrides.
+   */
+  protected function setLockupOverrides(array $names, array &$overrides) {
+
+    // Avoid circular loops.
+    if (!$this->configFactory || in_array('system.theme', $names)) {
+      return;
+    }
+
+    // Validate we are working with the info we need.
+    $theme_info = $this->configFactory->get('system.theme');
+    if (!$this->configPagesLoader || !$theme_info || !in_array($theme_info->get('default') . '.settings', $names)) {
+      return;
+    }
+
+    // Get the default theme from the config.
+    $theme_name = $theme_info->get('default');
+    $config_page = $this->configPagesLoader->load('lockup_settings');
+
+    // Failed to load the config page or not enabled.
+    if (!$config_page || $config_page->get('su_lockup_enabled')->getString() == "1") {
+      return;
+    }
+
+    // Do the overrides.
+    $this->setLockupTextOverrides($overrides, $theme_name, $config_page);
+
+    // Get and set a file path that is relative to the site base dir.
+    $this->setLockupFileOverrides($overrides, $theme_name, $config_page);
+
+  }
+
+  /**
+   * Set the lockup text overrides.
+   *
+   * @param array $overrides
+   *   The array of overrides.
+   * @param string $theme_name
+   *   The name of the default theme.
+   * @param \Drupal\config_pages\ConfigPagesInterface $config_page
+   *   A config page object.
+   */
+  protected function setLockupTextOverrides(array &$overrides, $theme_name, ConfigPagesInterface $config_page) {
+    $overrides[$theme_name . '.settings']['lockup']['option'] = $config_page->get('su_lockup_options')->getString();
+    $overrides[$theme_name . '.settings']['lockup']['line1'] = $config_page->get('su_line_1')->getString();
+    $overrides[$theme_name . '.settings']['lockup']['line2'] = $config_page->get('su_line_2')->getString();
+    $overrides[$theme_name . '.settings']['lockup']['line3'] = $config_page->get('su_line_3')->getString();
+    $overrides[$theme_name . '.settings']['lockup']['line4'] = $config_page->get('su_line_4')->getString();
+    $overrides[$theme_name . '.settings']['lockup']['line5'] = $config_page->get('su_line_5')->getString();
+    $overrides[$theme_name . '.settings']['use_logo'] = ($config_page->get('su_use_theme_logo')->getString() == "1") ? TRUE : FALSE;
+    $overrides[$theme_name . '.settings']['logo']['use_default'] = $overrides[$theme_name . '.settings']['use_logo'];
```

Comment:
in config overrides the `$overrides` array is empty. you can do normal array construction. 
```
$overrides[$theme_name . '.settings'] = [
...
];
```